addon/templates/components/frost-select-dropdown.hbs

```diff
@@ -63,12 +63,22 @@
             size='medium'
           }}
         {{/if}}
-        <span
+        <div class="frost-select-list-item-container">
```

Comment:
Might be a github thing but looks like the indentation is off between this div and the div below